lib/food-event-listeners.js

```diff
@@ -0,0 +1,3 @@
+const addNewFood = require('./index')
+
+$('#submit').on('click', addNewFood)
```

Comment:
Believe this code is duplicated in your `index.js` file.

Also, organization feels backwards to me. It seems like you have a lot going on in `index.js`. My instinct would be to treat that as a tool to bring in code from other files in an organized manner, but have most of the code itself in other files.

Would love to hear what @kat3kasper thinks, but it totally makes sense to me that you would have a `food-event-listeners.js` file, but that file should include the majority of the code that is required to add those event listeners. Then you bring that code into your `index.js` file, which is the file that you specified in `webpack.config.js` as the entry point for your code.

Reply:
Awesome, thanks Sal! @wiseemily88 and I are planning on finishing out our fetch calls (we have work on some separate branches at the moment), and then moving them all to an event listeners file.

Reply:
I agree with Sal. Typically the `index` file will be doing most of the imports. This is outdated and I'm not seeing this file in master. We will be doing a organizing client side code class on Tuesday so hopefully that will help with any questions you may have.